src/main/play-doc/operation/Licensing.md

```diff
@@ -0,0 +1,107 @@
+# Licensing
+
+It is required to load a license into your ConductR cluster if you want to be able to use more than one ConductR agent. 
+To license the cluster you must use the ConductR CLI `conduct load-license` command.
+
+The license will only needed to be loaded once for as long as the cluster is up, or until the license expires.  
+
+## Loading the License
+
+### Online Mode
+
+The standard workflow assumes that the workstation you are running the CLI on has connectivity to the internet.  To load
+the license 
+
+1. Execute `conduct load-license` at the command line.
+2. Retrieve your Enterprise Suite access token from https://www.lightbend.com/platform/enterprise-suite/access-token
+3. Enter your access token when prompted
+
+```
+$ conduct load-license
+
+An access token is required. Please visit https://www.lightbend.com/platform/enterprise-suite/access-token to 
+obtain one for free or for your commercial licenses.
+
+Please enter your access token: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+Loading license into ConductR at 192.168.10.1
+
+Licensed To: aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa
+Expires In: 365 days (Fri 27 Jul 2018 16:09PM)
+Grants: akka-sbr, cinnamon, conductr, fortify
+
+License successfully loaded
+```
+
+If successful you will see the following
+- A message indicating the license was added to the ConductR cluster.
+- A summary of license information, including your user token.  This will match your bintray credentials in your 
+`~/.lightbend/commercial.properties` file.
+- A date when the license expires and you will need to execute `load-license` again.
+- Components of Enterprise Suite that your license is authorized for.
+
+### Offline Mode
+
+In privileged environments, loading the license is straightforward.  In environments that cannot access the internet you 
+must install the license in offline mode (i.e. from your CLI without having the HTTP proxy setup, or if you’re 
+installing the CLI on a server with no internet access).
+
+Steps to install the license in offline mode.
+
+1. Login to your lightbend.com account with your licensed email and setup your `~/.lightbend/commercial.credentials` 
+as described on [lightbend.com](https://www.lightbend.com/product/conductr/developer).  While setting up your 
+`commercial.credentials` it’s important to use the email associated with your organization's license, and not a personal
+account.  If enterprise environments this is usually your corporate email.
+2. Download your `license` file manually.  You can find your license by visiting 
+https://www.lightbend.com/product/conductr/license
```

Comment:
That particular page was created for the HPE Fortify Scala plugin feature. The verbiage on that page can certainly be improved. Particularly now that we've added another use-case.